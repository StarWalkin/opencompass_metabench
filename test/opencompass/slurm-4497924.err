  0%|          | 0/1 [00:00<?, ?it/s]                                       0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [03:42<00:00, 222.67s/it]100%|██████████| 1/1 [03:42<00:00, 222.69s/it]
  0%|          | 0/1 [00:00<?, ?it/s]                                       0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:12<00:00, 12.11s/it]100%|██████████| 1/1 [00:12<00:00, 12.11s/it]
