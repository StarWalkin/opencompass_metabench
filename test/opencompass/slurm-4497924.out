/u/zzhang27/.conda/envs/opencompass/bin/python3
08/17 02:34:57 - OpenCompass - INFO - Loading gsm8k_gen: configs/datasets/gsm8k/gsm8k_gen.py
08/17 02:34:57 - OpenCompass - INFO - Loading hf_phi_3_small_8k_instruct: configs/models/phi/hf_phi_3_small_8k_instruct.py
08/17 02:34:57 - OpenCompass - INFO - Loading example: configs/summarizers/example.py
08/17 02:34:57 - OpenCompass - INFO - Current exp folder: outputs/default/20240817_023457
08/17 02:34:57 - OpenCompass - WARNING - SlurmRunner is not used, so the partition argument is ignored.
08/17 02:34:57 - OpenCompass - INFO - Partitioned into 1 tasks.
launch OpenICLInfer[phi-3-small-8k-instruct-hf/gsm8k] on GPU 0
08/17 02:38:40 - OpenCompass - ERROR - /scratch/bcwg/Metabench/repo/test/opencompass/opencompass/runners/local.py - _launch - 228 - task OpenICLInfer[phi-3-small-8k-instruct-hf/gsm8k] fail, see
outputs/default/20240817_023457/logs/infer/phi-3-small-8k-instruct-hf/gsm8k.out
08/17 02:38:40 - OpenCompass - ERROR - /scratch/bcwg/Metabench/repo/test/opencompass/opencompass/runners/base.py - summarize - 64 - OpenICLInfer[phi-3-small-8k-instruct-hf/gsm8k] failed with code 1
08/17 02:38:40 - OpenCompass - INFO - Partitioned into 1 tasks.
launch OpenICLEval[phi-3-small-8k-instruct-hf/gsm8k] on CPU 
dataset    version    metric    mode    phi-3-small-8k-instruct-hf
---------  ---------  --------  ------  ----------------------------
gsm8k      -          -         -       -
08/17 02:38:52 - OpenCompass - INFO - write summary to /scratch/bcwg/Metabench/repo/test/opencompass/outputs/default/20240817_023457/summary/summary_20240817_023457.txt
08/17 02:38:52 - OpenCompass - INFO - write csv to /scratch/bcwg/Metabench/repo/test/opencompass/outputs/default/20240817_023457/summary/summary_20240817_023457.csv
All processes have been started and their PIDs stored in process_ids_gpt4o.txt.
All processes have completed.
